packages/state-transition/test/unit/stateTransition.test.ts

```diff
@@ -1,48 +0,0 @@
-import fs from "node:fs";
```

Comment:
this test was based on beacon state structure from devnet1, since it has changed, is no longer relvant here